site/components/src/main/resources/META-INF/hst-assembly/overrides/valueListManager.xml

```diff
@@ -15,48 +15,34 @@
   See the License for the specific language governing permissions and
   limitations under the License.
   -->
```

Comment:
Looks like many entries in this file has got removed including the order of entries. Could we put them back and just include entries for `publicationTypes`, `publicationProfessions` and `publicationProfessions ` for clarity?